plugin.py

```diff
@@ -9,16 +12,20 @@
         'this package.'
     )
     sublime.error_message(" ".join(fmt).format(name))
+elif sublime.version() < '4000':
```

Comment:
I can't mark the code above but there is `__package__` that can be used instead of `os.path.basename(os.path.dirname(__file__))`.